server/jetty-app/README.md

```diff
@@ -1,70 +1,29 @@
-If you are interested in running a native production release, please see
+If you are interested in running a native production release or to see other options for starting the server, please see
 [https://deephaven.io/core/docs/how-to-guides/configuration/native-application/](https://deephaven.io/core/docs/how-to-guides/configuration/native-application/).
 
-### Groovy Quickstart
-
-```shell
-./gradlew server-jetty-app:run -Pgroovy 
-```
-
-### Python Quickstart
-
-1. Setup virtual environment:
-
-   ```shell
-   python -m venv /tmp/my-dh-venv
-   source /tmp/my-dh-venv/bin/activate
-   ```
-
-1. Build and install wheel
-
-   ```shell
-   ./gradlew py-server:assemble
+If using Python, see [Python development guide](../../py/README.md) for setting up the Python environment.
 
-   # replace with the appropriate <version>
-   pip install "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"
+## Local Development
 
-   # To install without the optional `autocomplete` feature, run:
-   # pip install "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl"
-   ```
+`server-jetty-app:run` will incorporate local Java changes on each start. If you are not frequently changing Java code, see the next section.
 
-1. Run
-
-   ```shell
-   ./gradlew server-jetty-app:run
-   ```
-
-**Note:**
-
-* This is not an exhaustive guide to managing python environments
-* Depending on your OS and how your PATH is setup, you may need to use `python3`, or a path to the explicit python version you want to use
-* You may choose to setup a "permanent" virtual environment location
-* You'll need to re-install the wheel anytime you are making python code changes that affect the wheel
-* `pip` can be a pain if you are trying to (re-)install a wheel with the same version number as before
-  * A `pip install --force-reinstall --no-deps "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"` may do the trick
-* You can install other python packages in your venv using `pip install <some-other-package>`
-* You can setup multiple virtual environments, and switch between them as necessary using `source /path/to/other-venv/bin/activate`
-* You can de-activate the virtual environment by running `deactivate`
-* You can use the `VIRTUAL_ENV` environment variable instead of sourcing / activating virtual environments: `VIRTUAL_ENV=/my/venv ./gradlew server-jetty-app:run`
+```shell
+./gradlew server-jetty-app:run # Python enigne by default
+./gradlew server-jetty-app:run -Pgroovy # Groovy engine
+```
 
-### Start script
+## Development with Infrequent Changes
 
-To create a more production-like environment, you can create and invoke the start script instead of running via gradle:
+To create a more production-like environment, you can create and invoke the start script instead of running via gradle. This is faster if you need to often restart the server without making any changes to Java code (such as Python server development).
```

Comment:
Again, we are discussing Python, when Python people were directed away in an early step.